Snakefile

```diff
@@ -0,0 +1,197 @@
+import pandas as pd
```

Comment:
Looking forward to the snakemake mini-workshop next week. This reads so much easier than a lot of Nextflow things le sigh